pkg/templates/latesttag/internal/params/params.go

```diff
@@ -5,4 +5,7 @@ type Params struct {
 
 	// list of regular expressions for blocked or bad container image tags
 	BlockList []string
+
+	// list of regular expressions for allowed container image conventions
+	AllowList []string
```

Comment:
Please update the comments to specify what happens if allowed and blocked are both specified (ie, blocked takes precedence), and what happens if `AllowList` is empty (that it means everything is allowed)

Reply:
Added the above specifications in the check description. 

Looks like [following]( https://github.com/stackrox/kube-linter/blob/c65dd74013ef482ed9ff2f8fcce4dbf926a74fae/docs/generated/templates.md#latest-tag):

![Screenshot from 2021-08-12 07-20-52](https://user-images.githubusercontent.com/30499743/129126164-62cb9a9e-95b1-4555-a4c1-c1f6084c998f.png)